src/services/HighScoreServiceFactory.ts

```
@@ -0,0 +1,36 @@
+    }
+
+    // Use remote service for production or properly configured development
+    try {
+      console.log('Using HighScoreService with Supabase');
+      return new HighScoreService();
+    } catch (error) {
+      console.warn('Failed to create HighScoreService, falling back to LocalHighScoreService:', error);
+      return new LocalHighScoreService();
+    }
+  }
+} 
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)